stacks_queues/stack.py

```diff
@@ -12,7 +12,13 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+
+        self.store.add_last(element)
+        # if not self.store.head:
+        #     self.store.add_first(element)
+        #     return
+        # self.store.tail.next = element
+        # self.store.tail = element
 
     def pop(self):
         """ Removes an element from the top
```

Comment:
✨ This will work, however you could also take advantage of the `remove_last` method in the `LinkedList` class